dev/how-to/maintain/identify-slow-queries.md

```diff
@@ -35,21 +30,38 @@ select count(1) from t_slim, t_wide where t_slim.c0>t_wide.c0 and t_slim.c1>t_wi
 
 ## Fields description
 
+> **Note:**
+>
+> The unit of all the following time fields in the slow query log is **"second"**.
+
+Slow Query Basics:
+
 * `Time`: The print time of log.
-* `Txn_start_ts`: The start timestamp of the transaction (transaction ID). You can use this value to `grep` the transaction-related logs.
+* `Query_time`: Indicates the execution time of this statement.
+* `Query`: A SQL statement. `Query` is not printed in the slow log, but the corresponding field is called `Query` after the slow log is mapped to the memory table.
+* `Digest`: The fingerprint of the SQL statement.
+* `Txn_start_ts`: The start timestamp and the unique ID of the transaction. You can use this value to `grep` the transaction-related logs.
+* `Is_internal`: Whether the SQL statement is TiDB internal. `true` indicates that the SQL statement is executed internally in TiDB, while `false` indicates that the SQL statement is executed by the user.
+* `Index_ids`: The IDs of the indexes involved in the statement.
+* `Succ`: Whether the statement executes successfully.
+* `Backoff_time`: The waiting time before retry when this statement encounters errors that require a retry. The common errors as such include: `lock occurs`, `Region split`, and `tikv server is busy`.
+
+Memory usage fields:
+
+* `Memory_max`: The maximum memory space used during the execution period of this SQL statement (the unit is byte).
```

Comment:
I don't think "this" or ''the" is accurate here because the fields do not refer to a specific SQL statement.
```suggestion
* `Memory_max`: The maximum memory space used during the execution period of a SQL statement (the unit is byte).
```